### ISSUE TITLE
Establish rules for fonts within input fields and apply consistently.

### ISSUE THREAD

**Issue by Skaiir** [bpmn-io/form-js#506]
### What should we do?

Currently, there is a discrepancy between fonts within inputs and fonts in the general UI. Fonts in inputs are sized to 13.3333px while fonts outside of it are sized to 14px. There are also some further inconsistencies in terms of disabled font color, and styling of fonts in these states. Styling of placeholders, etc. 

Let's carry out a discussion on what we want these things to look like, establish some rules and clean up across all components. The implementation itself should be rather trivial, but there's certainly some back and forth that can occur in terms of decision-making.  

For reference here is what an input field looks like currently at 13+1/3px
![image](https://user-images.githubusercontent.com/17801113/213705080-8a087db3-5e11-4566-a4bd-7bf7c52ed03b.png)

And how it would look with a 14px font size
![image](https://user-images.githubusercontent.com/17801113/213704986-09d54c82-d234-438d-a94e-45e12f32221c.png)

### Why should we do it?

To offer a consistent UX across components, and if relevant in this scenario to keep the readability of our components high (a11y)

**Comment by Skaiir** [comment]
@nikku @RomanKostka @christian-konrad

Would love to gather some thoughts here on what we think this should look like. 

The way carbon handles this, they have 12px labels and 14px inputs. The 13.33px text is definitely at the root of a few off-by-one pixel errors, but having the exact same size for labels and input fields is a bit heavy on the eyes (at least for me), as it makes it harder to know what to focus on. 

Perhaps following carbon on this one may be the right thing. But I'm very open to opinions here. I'll post a few screenshots of what different styling look like in a bit.

**Comment by Skaiir** [comment]
Input 14px (I count the radio values as inputs rather than labels)
Descriptions + Label 13

![image](https://user-images.githubusercontent.com/17801113/214823001-43f6813e-3a8b-4530-912b-2002fba01c64.png)

14px for both

![image](https://user-images.githubusercontent.com/17801113/214823323-d8e59309-b086-4955-a3c9-f930303ec3b9.png)

14px / 12px

![image](https://user-images.githubusercontent.com/17801113/214824004-a8b1bb6a-bbb6-4a54-aeca-da84a6c61502.png)


I haven't adjusted margins, but personally, I find 14/13 to be the most comfortable.

**Comment by nikku** [comment]
:+1: for "being same size is heavy on the eyes". I think there is good reasons that inputs (I enter) and labels operate at different visual importance.

Other ways to accomplish that exist, of course, i.e. through coloring.

I'd honestly think it is worth to simply adopt what Carbon does (sizing, but also coloring).

**Comment by RomanKostka** [comment]
totally agree!
i would also propose to stick to carbon guidelines here 👍

**Comment by vsgoulart** [comment]
@Skaiir You can find the typography docs for Carbon [here](https://carbondesignsystem.com/guidelines/typography/type-sets). For components they `body-compact-01` for the input and `label-01` for the helper texts.
